app/views/users/_meta.html.erb

```diff
@@ -1,19 +1,19 @@
 <link rel="canonical" href="<%= user_url(@user) %>" />
-<meta name="description" content="<%= @user.profile.summary %>">
+<meta name="description" content="<%= @user.tag_line %>">
 <%= meta_keywords_default %>
 
 <meta property="og:type" content="website" />
 <meta property="og:url" content="<%= user_url(@user) %>" />
 <meta property="og:title" content="<%= @user.name %> — <%= community_name %> Profile" />
 <meta property="og:image" content="<%= user_social_image_url(@user) %>">
-<meta property="og:description" content="<%= @user.profile.summary %>" />
+<meta property="og:description" content="<%= @user.tag_line %>" />
 <meta property="og:site_name" content="<%= community_name %>" />
-
+    <meta name="twitter:card" content="summary_large_image">
```

Comment:
isn't this the same as the following line?

Reply:
Yes, but the indentation is different. Maybe a merge artifact, thanks!